packages/jellyfish-api-core/src/category/poolpair.ts

```diff
@@ -120,6 +120,23 @@ export class PoolPair {
   async testPoolSwap (metadata: TestPoolSwapMetadata): Promise<string> {
     return await this.client.call('testpoolswap', [metadata], 'bignumber')
   }
+
+  /**
+   * Remove pool liquidity transaction
+   *
+   * @param {RemovePoolLiquiditySource} from pool liquidity sources
+   * @param {string | string[]} from[address] provides at least two types of token with format 'amoun@token'
```

Comment:
Should remove. As you do not pass this parameter in your function

Reply:
Got it, I'll make the changes.

Reply:
@jingyi2811 I removed the lines as suggested and replaced with 
```
 * @param {string} address defi address for crediting tokens
 * @param {string} poolAccount pool liquidity account of owner
```

as there are new params in the test function, but not sure why it's not reflecting here. 🤔